From f82607dd655fb3609b190bb452ab5592c1b01000 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 19 Jun 2014 19:19:44 +0800
Subject: [PATCH] ENGR00319455-4 imx_v7_defconfig: enable canfd by default

enable canfd by default.
The defconfig is generated by make ARCH=arm savedefconfig
which got the uneeded CONFIG_USB_PHY removed.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit 0d6fbbce14deea6faba1a6dc9b9866b70a0e4c2f)
---
 arch/arm/configs/imx_v7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 86a1b2d..663ae71 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -67,6 +67,7 @@ CONFIG_NETFILTER=y
 CONFIG_VLAN_8021Q=y
 CONFIG_CAN=y
 CONFIG_CAN_FLEXCAN=y
+CONFIG_CAN_M_CAN=y
 CONFIG_CFG80211=y
 CONFIG_MAC80211=y
 CONFIG_RFKILL=y
-- 
1.8.0

